bin/improver-wind-gust-diagnostic

```diff
@@ -0,0 +1,88 @@
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to create wind-gust data."""
+
+import argparse
+
+import iris
+import warnings
+
+from improver.wind_gust_diagnostic import WindGustDiagnostic
+
+
+def main():
+    """Load in arguments for wind-gust diagnostic.
+    Wind-gust and Wind-speed data should be supplied along with the required
+    percentile value. The wind-gust diagnostic will be the Max of the specified
+    percentile data.
+    Currently
+        * Typical gusts is
+          MAX(wind-gust(50th percentile),wind-speed(95th percentile))
+        * Extreme gust is
+          MAX(wind-gust(95th percentile),wind-speed(100th percentile))
+    If no percentile values are supplied the code defaults
+    to values for Typical gusts.
+    """
+    parser = argparse.ArgumentParser(
+        description="Calculate revised wind-gust data using a specified "
+        "percentile of wind-gust data and a specified percentile "
+        "of wind-speed data through the WindGustDiagnostic plugin.")
+    parser.add_argument("input_filegust", metavar="INPUT_FILE_GUST",
+                        help="A path to an input Wind Gust Percentile"
+                        " NetCDF file")
+    parser.add_argument("input_filews", metavar="INPUT_FILE_WINDSPEED",
+                        help="A path to an input Wind Speed Percentile"
+                        " NetCDF file")
+    parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
+                        help="The output path for the processed NetCDF")
+    parser.add_argument("--percentile_gust", metavar="PERCENTILE_GUST",
+                        default="50.0",
+                        help="Percentile of wind-gust required."
+                        " Default=50.0")
+    parser.add_argument("--percentile_ws",
+                        metavar="PERCENTILE_WIND_SPEED",
+                        default="95.0",
+                        help="Percentile of wind-speed required."
+                        " Default=95.0")
+
+    args = parser.parse_args()
+    cube_wg = iris.load_cube(args.input_filegust)
+    cube_ws = iris.load_cube(args.input_filews)
+    result = (
+        WindGustDiagnostic(float(args.percentile_gust),
```

Comment:
The argparser could deal with the float bit to make this cleaner, simply define a type in the relevant add_argument statements.